linkinator.config.json

```diff
@@ -2,7 +2,6 @@
   "recurse": true,
   "skip": [
     "https://codecov.io/gh/googleapis/",
-    "www.googleapis.com",
-    "img.shields.io"
```

Comment:
The PR to include this landed recently in the generator - would be nice to keep this :)

Reply:
cool, thanks!